src/Assembly.js

```diff
@@ -347,12 +390,17 @@ class Assembly extends React.Component {
           {() => React.createElement(this.currentPage, { store: this })}
         </Observer>
       </Content>
-
+      
       <Space />
-
-      <NavBar>
-        <Navigation store={this} />
-      </NavBar>
+      {/* Only shows the navbar when a user is logged in */}
+      <Observer>
+        {() => 
+          this.registration.information.uuid == null ? null : 
+            <NavBar>
+              <Navigation store={this} />
+            </NavBar>
+        }
+        </Observer>
```

Comment:
This looks good, but we'll likely want to apply the same logic to the `menu` element above.
Can you work some copy/paste magic?

Reply:
Good catch